src/f_finale.c

```diff
@@ -150,7 +300,7 @@ static boolean MapInfo_Ticker()
 
         if (finalestage == FINALE_STAGE_CAST)
         {
-            F_CastTicker();
+            next_level = F_CastTicker();
```

Comment:
You immdeatly return from the function, so assigment  of `next_level` variable is not needed.